### ISSUE TITLE
Just creating new File object does not create a new File in a certain directory

### ISSUE THREAD

**Issue by sandydays** [nus-cs2103-AY1920S1/forum#27]
I tried this example on my desktop:

![image](https://user-images.githubusercontent.com/50706672/63935299-bb1ec200-ca8f-11e9-9e56-90c7bc253b2d.png)
When I run this Java program, I get this result instead:
```
file exists?: false
is Directory?: false
```

The file isn't created just by creating a new instance of the `File` object. As a result, I am also having trouble implementing `Level-7`. Any help as to how we can create a new file?

**Comment by dawo5010** [comment]
Heyyo,

I had the same issue when trying to complete Level-7 as well, and I didn't want to make it such that the file had to pre-exist/be created by the user before using the program.

Here's some articles on stackoverflow that I referred to for this problem: [Create File if not exists](https://stackoverflow.com/questions/9620683/java-fileoutputstream-create-file-if-not-exists), [Create whole path automatically](https://stackoverflow.com/questions/2833853/create-whole-path-automatically-when-writing-to-a-new-file).

In short, use .getParentFile().mkdirs() on your file object to create the necessary directory, and use .createNewFile() to create the file itself. Hope that helps!

**Comment by damithc** [comment]
@sandydays resolved?

**Comment by sandydays** [comment]
Yes, it has been resolved! Thanks a lot for the help @dawo5010 and sorry for replying back to this a bit late!
